src/bedrock/route/routing.cpp

```diff
@@ -19,6 +19,12 @@ std::unordered_map<unsigned, TierData> kTierDataMap;
 unsigned kDefaultLocalReplication;
 unsigned kRoutingThreadCount;
 
+ZmqUtil zmq_util;
```

Comment:
It seems bad to declare a global variable and then assign a pointer to another global variable. Am I being overly nitpicky? Can we thin of a less gross way to do this?

Reply:
We can't do things like
`ZmqUtilInterface kZmqUtil = &ZmqUtil()`, as it will complain assigning an address to temp variable.
I tried to not make `kZmqUtil` a pointer, but that doesn't work either because we can't instantiate an abstract class in C++.

Reply:
Makes sense. Thanks for checking.